api/app/Models/User.php

```diff
@@ -913,6 +913,21 @@ public static function scopeGeneralSearch(Builder $query, ?array $searchTerms):
                 // Now that we have added a column, query builder no longer will add a * to the select.  Add all possible columns manually.
                 ->addSelect(self::$selectableColumns);
 
+            // clear characters or search operators out, then create array for existing scopes by splitting into words
+            $filterToEmptySpace = ['"', '"', '-', ':', '!'];
+            $filterToSingleSpace = [' AND ', ' OR '];
+            $filtered = str_ireplace($filterToEmptySpace, '', $combinedSearchTerm);
+            $filtered = str_ireplace($filterToSingleSpace, ' ', $filtered);
+            $arrayed = explode(' ', $filtered);
```

Comment:
Why did you split this into an array? The name scope can already handle multiple words.

Reply:
Huh, it turns out it was totally unneeded 😅 
Got rid of it in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9977/commits/5a2a5d68e66e889f563d8fea63f094141da38b2e

Reply:
Or maybe it did help...
😅 🤷

Reply:
Yeah, now I can see it helps with the expanded test cases